spec/build/script/csharp_spec.rb

```diff
@@ -13,6 +13,7 @@
 
   describe 'configure' do
     it 'sets up package repository' do
+      data[:config][:os] = 'linux'
```

Comment:
these changes tell me that we need to add a default value for os=linux, otherwise every existing .travis.yml out there will break